### ISSUE TITLE
[aria-*] attributes do not have valid values. On select component

### ISSUE THREAD

**Issue by PringlePot** [mantinedev/mantine#1591]
### What package has an issue

@mantine/core

### Describe the bug

The select component will have aria-activedescendant, set when prerendered with nextjs. This issue doesn't really affect the user experience. However lighthouse will complain about the value not being a valid div. When hovering, and then hovering off the select will cause it to disapear. When you hover over the component in the codesandbox the aria field will dissapear

### In which browser did the problem occur

Google Chrome

### If possible, please include a link to a codesandbox with the reproduced problem

https://codesandbox.io/s/old-cherry-95syv1?file=/pages/index.tsx

### Do you know how to fix the issue

No

### Are you willing to participate in fixing this issue and create a pull request with the fix

Yes

### Possible fix
Currently aria-activedescendant, is set when hovered isn't -1. So we might be able to set hovered to -1 as default, to prevent nextjs from adding the  aria-activedescendant. Or we could check if hovered != null, since that is the case for the ssr'ed page
https://github.com/mantinedev/mantine/blob/340adf8af50a5d37ee5d12cc0e8c2d2bf88a5ee0/src/mantine-core/src/components/Select/Select.tsx#L600

The usestate is already set to -1 by default tho
https://github.com/mantinedev/mantine/blob/340adf8af50a5d37ee5d12cc0e8c2d2bf88a5ee0/src/mantine-core/src/components/Select/Select.tsx#L240

So checking if it isnt null might be the best option?

**Comment by wes337** [comment]
Could you help me to better understand this issue?

When you focus on an item in the Select component, the `aria-activedescendant` attribute is populated:
![image](https://user-images.githubusercontent.com/9464690/172985423-835a0d22-b431-4220-9669-e7ddea10f170.png)

When nothing is focused, the attribute is removed.

This is the correct behavior. I am also confused how this is specific to Next.js?

**Comment by rtivital** [comment]
We've discussed this on Discord, the issue is that `aria-activedescendant` attribute is added to the input element on initial render (before dropdown is opened), after user interacts with the input `aria-activedescendant` works as expected.

**Comment by PringlePot** [comment]
I have seem to have narrowed down the issue. It happens when the select is controlled, and the initial value is null.

**Comment by wes337** [comment]
> We've discussed this on Discord, the issue is that `aria-activedescendant` attribute is added to the input element on initial render (before dropdown is opened), after user interacts with the input `aria-activedescendant` works as expected.

Can't reproduce this.

Here's the elements right on initial render, before any interaction: 
![image](https://user-images.githubusercontent.com/9464690/173173457-4d756691-f41e-40aa-b37a-e7f352a6103d.png)

There's no `aria-activedescendant`. Checked on both Firefox and Chrome. Am I still missing something?

**Comment by wes337** [comment]
But if the fix is as @PringlePot says, then it would probably just be:
```jsx
aria-activedescendant={hovered >= 0 ? `${uuid}-${hovered}` : null}
```

Instead of checking that it doesn't equal -1.

**Comment by PringlePot** [comment]
That might fix the issue, I'll try to find an example that has the issue 100%

**Comment by rtivital** [comment]
Fixed in 4.2.10
